software/tuxseo.yml

```
@@ -0,0 +1,14 @@
+name: TuxSEO
+website_url: https://tuxseo.com/
+source_code_url: https://github.com/rasulkireev/TuxSEO
+description: "Create automated blog content for your business, using AI."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Django
+  - Docker
+tags:
+  - Blogging Platforms
+  - Generative Artificial Intelligence (GenAI)
```

```suggestion
tags:
  - Generative Artificial Intelligence (GenAI)
  - Blogging Platforms
```

> https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#add-software-to-the-list

>> In [single page mode](https://github.com/awesome-selfhosted/awesome-selfhosted) the software will only appear under the first category in its tags list, so choose wisely.

Since the main use case is not blogging but generating AI slop, I think the order should be inverted.

---

@nodiscc , makes sense. updated.